app/models/classroom.rb

```diff
@@ -26,4 +26,11 @@ class Classroom < ApplicationRecord
   has_many :users, through: :student_classrooms
 
   validates :name, presence: true
+  validate :is_teacher, on: :create 
+ 
+  def is_teacher
```

Comment:
consider renaming the method or rewriting the logic
as it is currently, it returns true when the user is admin, which doesn't make sense